src/main/java/io/cdap/plugin/servicenow/apiclient/ServiceNowTableAPIClientImpl.java

```diff
@@ -147,19 +145,21 @@ public List<Map<String, Object>> fetchTableRecords(String tableName, SourceValue
    * @param tableName ServiceNow Table name
    * @param entity    Details of the Record to be created
    */
-  public void createRecord(String tableName, HttpEntity entity) throws IOException {
+  public String createRecord(String tableName, HttpEntity entity) throws IOException {
```

Comment:
Annotate with @VisibleForTesting

Reply:
I am actually using this method in the **e2e-test** package.(If I will use the Annotation Ill be only able to use it in Test classes [Junit tests classes]).